messaging/src/main/java/org/axonframework/eventhandling/processors/streaming/token/store/jpa/JpaTokenStore.java

```diff
@@ -106,29 +97,45 @@ protected JpaTokenStore(Builder builder) {
         this.loadingLockMode = builder.loadingLockMode;
     }
 
-    @Override
-    public void initializeTokenSegments(@Nonnull String processorName, int segmentCount)
-            throws UnableToClaimTokenException {
-        initializeTokenSegments(processorName, segmentCount, null);
+    /**
+     * Instantiate a Builder to be able to create a {@code JpaTokenStore}.
+     * <p>
+     * The {@code claimTimeout} to a 10 seconds duration, and {@code nodeId} is defaulted to the name of the managed
+     * bean for the runtime system of the Java virtual machine. The {@link EntityManagerProvider} and {@link Serializer}
+     * are a <b>hard requirements</b> and as such should be provided.
+     *
+     * @return a Builder to be able to create a {@code JpaTokenStore}
+     */
+    public static Builder builder() {
```

Comment:
Errr, darn, I think we should get rid of these builders too... That can be a follow-up, perhaps something I or someone else does quickly. Should've spotted this sooner!